package.json

```diff
@@ -1,97 +1,38 @@
 {
-  "name": "@metamask/controllers",
+  "name": "@metamask/controllers-monorepo",
   "version": "32.0.2",
+  "private": true,
   "description": "Collection of platform-agnostic modules for creating secure data models for cryptocurrency wallets",
-  "keywords": [
-    "MetaMask",
-    "Ethereum"
-  ],
-  "homepage": "https://github.com/MetaMask/controllers#readme",
-  "bugs": {
-    "url": "https://github.com/MetaMask/controllers/issues"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/MetaMask/controllers.git"
   },
-  "license": "MIT",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "files": [
-    "dist/"
+  "workspaces": [
+    "packages/*"
   ],
   "scripts": {
-    "build": "rimraf dist && tsc --project tsconfig.build.json",
-    "build:link": "yarn build && cd dist && yarn link && rm -rf node_modules && cd ..",
-    "build:watch": "yarn build --watch",
-    "doc": "typedoc && touch docs/.nojekyll",
-    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "build": "tsc --build tsconfig.build.json --verbose",
+    "build:clean": "rimraf './**/*.tsbuildinfo' './dist'",
+    "build:watch": "yarn run build --watch",
+    "changelog:validate": "yarn workspaces foreach --parallel --interlaced --verbose run changelog:validate",
+    "doc": "yarn workspaces foreach --parallel --interlaced --verbose run doc",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn constraints",
     "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '!**/CHANGELOG.old.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore",
     "prepack": "./scripts/prepack.sh",
     "setup": "yarn install",
-    "test": "jest",
-    "test:debug": "yarn run --inspect-brk jest --runInBand",
-    "test:watch": "jest --watch"
+    "test": "yarn workspaces foreach --verbose run test"
   },
   "simple-git-hooks": {
     "pre-push": "yarn lint"
   },
   "dependencies": {
-    "@ethereumjs/common": "^2.3.1",
-    "@ethereumjs/tx": "^3.2.1",
-    "@ethersproject/abi": "^5.7.0",
-    "@ethersproject/contracts": "^5.7.0",
-    "@ethersproject/providers": "^5.7.0",
-    "@keystonehq/metamask-airgapped-keyring": "^0.6.1",
-    "@metamask/contract-metadata": "^1.35.0",
-    "@metamask/metamask-eth-abis": "3.0.0",
-    "@metamask/types": "^1.1.0",
-    "@types/uuid": "^8.3.0",
-    "abort-controller": "^3.0.0",
-    "async-mutex": "^0.2.6",
-    "babel-runtime": "^6.26.0",
```

Comment:
This needs to be added back to any package that uses `ethjs-query`. See: https://github.com/MetaMask/controllers/pull/341

Reply:
Actually wait, this is there in the transactions controller but it's still needed in the assets, gas, and network controller

Reply:
Fixed here: https://github.com/MetaMask/controllers/pull/831/commits/b23a926a568ee825a72c75bd3ef40fc0b8156106